it/common.sh

```diff
@@ -32,7 +32,11 @@ fakeRelease() {
   ARCH=$(../../bin/go version | awk '{print $NF}' | cut -d/ -f2)
   mkdir -p "$DIR"
   gzip -c hermit > "$DIR/hermit-${OS}-${ARCH}.gz"
-  ../../.hermit/go/bin/geninstaller --dest="${DIR}/install.sh" --dist-url=-https://github.com/cashapp/hermit/releases/download/stable
```

Comment:
I'm not sure if the `-` before the URL string was intentional or not. I removed it so that it would match the URL used to create the installer hash, hard coded in `env.go`, and therefore would make the `Check install script SHA-256 sum` CI test pass.

Reply:
@alecthomas Do you see any problem if we remove the `-` here?

Reply:
I'm honestly not sure why that was there 🤔

Reply:
Oh I think because we don't actually want the integration tests to ever install from a remote URL @jvmakine ?